controllers/opsworks.py

```diff
@@ -1,27 +1,108 @@
+import sys
 
 import boto.opsworks
-import logging
+from boto.exception import JSONResponseError
 
 from ec2 import EC2Controller
-import settings
+from exceptions import *
 
+import logging
 log = logging.getLogger()
 
 class OpsworksController(EC2Controller):
 
-    def __init__(self, *args, **kwargs):
-        self._opsworks = None
-        super(OpsworksController, self).__init__(*args, **kwargs)
-
     @property
     def opsworks(self):
-        if self._opsworks is None:
-            self._opsworks = self.create_opsworks_connection()
+        if not hasattr(self, '_opsworks'):
```

Comment:
I'm just caching the client object. How boto handles the actual http connections (looks like a connection pool of some kind) isn't something I worried about here.